src/ExposureHistory/ExposureDetail.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
+import { Linking } from "react-native"
+import { render, cleanup, fireEvent } from "@testing-library/react-native"
+import { DateTimeUtils } from "../utils"
+import { factories } from "../factories"
+import ExposureDetail from "./ExposureDetail"
+import { useRoute } from "@react-navigation/native"
+
+afterEach(cleanup)
+jest.mock("react-native-config", () => {
+  return {
+    AUTHORITY_ADVICE_URL: "https://www.health.state.mn.us/",
+  }
+})
+
+jest.mock("@react-navigation/native")
+describe("ExposureDetail", () => {
+  describe("when the exposure happend in the last three days", () => {
+    it("formats the timeframe correctly", () => {
+      expect.assertions(1)
```

Comment:
Its not clear to me why we need these `expect.assertions` ? is there a async aspect to this? Or is this for better error messages?

Reply:
This is part of my UltiSnip because I usually enable a rule that requires you to expect assertions. Will remove!